Doc/library/stdtypes.rst

```diff
@@ -2760,12 +2762,14 @@ arbitrary binary data.
             bytearray.decode(encoding="utf-8", errors="strict")
 
    Return a string decoded from the given bytes.  Default encoding is
-   ``'utf-8'``. *errors* may be given to set a different
-   error handling scheme.  The default for *errors* is ``'strict'``, meaning
-   that encoding errors raise a :exc:`UnicodeError`.  Other possible values are
-   ``'ignore'``, ``'replace'`` and any other name registered via
-   :func:`codecs.register_error`, see section :ref:`error-handlers`. For a
-   list of possible encodings, see section :ref:`standard-encodings`.
+   ``'utf-8'``.  And for a list of possible encodings, see section
+   :ref:`standard-encodings`.
+
+   *errors* may be given to set a different error handling scheme.  The
+   default for *errors* is ``'strict'``, meaning that encoding errors raise a
+   :exc:`UnicodeError`.  Other possible values are ``'ignore'``, ``'replace'``
+   and any other name registered via :func:`codecs.register_error`, see
+   section :ref:`error-handlers`.
 
    By default, the *errors* argument is not checked for best performances, but
    only used at the first decoding error. Enable the :ref:`Python Development
```

Comment:
I was not able to actually make this as suggestion, but this implements the revision from the issue, while still including the key details from the previous version.

```rst
   For performance reasons, the value of *errors* is not checked for validity
   unless a decoding error actually occurs,
   :ref:`Python Development Mode <devmode>` is enabled
   or a :ref:`debug build <debug-build>` is used.
```